doc/api/assert.md

```diff
@@ -1419,6 +1419,59 @@ let err;
 //     at errorFrame
 ```
 
+## `assert.includes(string, substring[, message])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental.
+
+* `string` {string}
+* `substring` {string}
+* `message` {string|Error}
+
+Expects the `substring` input to be included in the `string` input.
+
+```mjs
+import assert from 'node:assert/strict';
+
+assert.includes('I will fail', 'pass');
+// AssertionError [ERR_ASSERTION]: The substring was not located inside the target string.
+
+assert.includes(123, 'pass');
+// AssertionError [ERR_ASSERTION]: The "string" argument must be of type string.
+
+assert.includes('I will fail', /pass/);
+// AssertionError [ERR_ASSERTION]: The "substring" argument must be of type string.
+
+assert.includes('I will pass', 'pass');
+// OK
+```
+
+```cjs
+const assert = require('node:assert/strict');
+
+assert.includes('I will fail', 'pass');
+// AssertionError [ERR_ASSERTION]: The substring was not located inside the target string.
+
+assert.includes(123, 'pass');
+// AssertionError [ERR_ASSERTION]: The "string" argument must be of type string.
+
+assert.includes('I will fail', /pass/);
+// AssertionError [ERR_ASSERTION]: The "substring" argument must be of type string.
+
+assert.includes('I will pass', 'pass');
```

Comment:
I think, it would be nice to add a valid case with for regexp, like:
```
assert.includes(/\bpass\b/, 'It will pass');
```
@zackschuster WDYT?

Reply:
that's just the `assert.match` api, but in reverse. i think that would be far more confusing than helpful.

Reply:
okay, for me word "includes" suitable to regexp matches. Well, with that point of view, it probably better to rename includes into strictIncludes, similar to [toEqual](https://jestjs.io/docs/expect#toequalvalue) and [toStrictEqual](https://jestjs.io/docs/expect#tostrictequalvalue)

Reply:
"strict equality" refers to a real JS thing (https://tc39.es/ecma262/#sec-isstrictlyequal), "strict includes" does not. I don't think we should go with `strictIncludes`.

> the word "includes" suitable to regexp matches

Arguably `match` is even more suitable.